charts/kubescape-cloud-operator/assets/host-scanner-definition.yaml

```diff
@@ -42,6 +42,7 @@ spec:
       - name: host-sensor
         image: quay.io/armosec/kube-host-sensor:latest
         securityContext:
+          allowPrivilegeEscalation: true
```

Comment:
Did it failed with "false"?

Reply:
We need to fix this file as well:
https://github.com/kubescape/kubescape/blob/master/core/pkg/hostsensorutils/hostsensor.yaml

Reply:
> Did it failed with "false"?
Nope, but I have a Kyverno mutation policy in place, which sets this field to `false` if its unset.

kubernetes PodSpec has implicit defaults:
> AllowPrivilegeEscalation is true always when the container is: 1) run as Privileged 2) has CAP_SYS_ADMIN

Maybe I can tweak the policy in place, but it also doesnt hurt, as privileged means AllowPrivilegeEscalation

Reply:
So let's set "allowPrivilegeEscalation": false, so we will have the minimun permissions required to run

Reply:
No see one line below:
`privileged: true` is already present.

Anf for `privileged: true` it implies that `allowPrivilegeEscalation: true`. I only set it explicitely, so no PSP/Kyverno/OPAGatekeeper will set this to `allowPrivilegeEscalation: false`.

PodSpec is only valid if `allowPrivilegeEscalation: true` AND `privileged: true`.

Reply:
Here's another project which prepared it inside the YAML:
https://github.com/kubereboot/charts/blob/c3656490312fb42ad79605cf3609ed0d55c9395e/charts/kured/values.yaml#L70-L72

Reply:
> We need to fix this file as well: https://github.com/kubescape/kubescape/blob/master/core/pkg/hostsensorutils/hostsensor.yaml

Also filed a PR over there: https://github.com/kubescape/kubescape/pull/976